tests.py

```diff
@@ -216,6 +216,60 @@ def test_create_or_update_create_time_catch_request_error(self, m):
                                                                "time",
                                                                "times"))
 
+    def test_create_or_update_time_with_junk_string_duration(self):
```

Comment:
So then:
```
    def test_create_or_update_time_with_invalid_string_duration(self):
        """Tests that TimeSync._create_time will fail if a string containing
        multiple hours/minutes is entered"""
        params = {
            "duration": "3h30m15h",
            "project": "ganeti-web-manager",
            "user": "example-user",
            "activities": ["documenting"],
            "notes": "Worked on docs",
            "issue_uri": "https://github.com/",
            "date_worked": "2014-04-17",
        }

>>>  # Mock requests.post so it doesn't actually post to TimeSync
>>>  requests.post = mock.create_autospec(requests.post)

>>>  self.ts._create_or_update(params, None, "time", "times")

        self.assertEquals(self.ts._create_or_update(params, None,
                                                    "time", "times"),
                          [{self.ts.error:
                            "time object: duration contains invalid string"}])
```
Do I need to @patch("pymesync.TimeSync._response_to_python") as well? I looked at the docs, but I'm still shaky on the specifics of mocking.